sdk/keyvault/TROUBLESHOOTING.md

```diff
@@ -58,10 +58,32 @@ See our [DefaultAzureCredential] documentation to see the order credentials are
 tenant for one credential that gets read before another credential. For example, you might be logged into Visual Studio
 under the wrong tenant even though you're logged into the Azure CLI under the right tenant.
 
+You can also explicitly set the tenant ID for a multi-tenant application if you reference `Azure.Identity` version
+1.5.0 or newer and any of the following Key Vault SDK package versions or newer:
```

Comment:
Wasn't this already the behavior available? I thought the new behavior to call out is the ability to not specify the tenant Id and have it auto-discovered via the challenge.

Reply:
Was that it? That info was so last year. I can change this, but this information isn't really wrong either. Should we just nix all the stuff about setting the tenant ID explicitly and mention auto-discovery then? Maybe Identity could document either approach. Maybe.

Reply:
Fixed with @schaabs help.